pkg/kubectl/stop.go

```diff
@@ -335,12 +335,16 @@ func (reaper *StatefulSetReaper) Stop(namespace, name string, timeout time.Durat
 	if err != nil {
 		return err
 	}
+
```

Comment:
I have a pr for removed this entire file here: https://github.com/kubernetes/kubernetes/pull/46927, ptal